test/validate/mcoe_test.py

```diff
@@ -91,27 +91,6 @@ def test_no_null_rows_mcoe(pudl_out_mcoe, live_dbs, df_name, thresh):
     )
 
 
-@pytest.mark.parametrize(
-    "df_name,unique_subset",
-    [
-        ("hr_by_unit", ["report_date", "plant_id_eia", "unit_id_pudl"]),
-        ("hr_by_gen", ["report_date", "plant_id_eia", "generator_id"]),
-        ("fuel_cost", ["report_date", "plant_id_eia", "generator_id"]),
-        ("capacity_factor", ["report_date", "plant_id_eia", "generator_id"]),
-        ("mcoe", ["report_date", "plant_id_eia", "generator_id"]),
```

Comment:
Verified that these are all PKs for the `PudlTabl` supporting underscore tables that are referenced by the short names here.